### ISSUE TITLE
Improving the accuracy

### ISSUE THREAD

**Issue by nikhil1205-ai** [androidvitb/Heart-disease-prediction-using-ECG#2]
Description :- This project employs machine learning techniques to analyze electrocardiogram (ECG) data for classifying heart conditions into four categories

Solution :-
1) we can use hyperparameters tuning techniques 
2) feature engineering and Normalize our data 
3)  Use the pre-trained CNN as a fixed feature extractor by freezing its convolutional layers and training only the fully connected layers (classifier) on the dataset.
4)  Ussing EfficientNet  ,  ResNet  , VGG Model

So  assign me this issue under acwoc

**Comment by androidvitb** [comment]
go ahead
